docs/usage.md

```diff
@@ -97,6 +97,42 @@ $image-path: 'img';
 @import 'identity-style-guide/dist/assets/scss/styles';
 ```
 
+If you're using Sprockets and precompiling assets you'll need to update your
+Sprockets manifest to include the images for production:
+
+```js
+// app/assets/config/manifest.js
+
+//= link_tree ../../../node_modules/identity-style-guide/dist/assets/img
```

Comment:
Since we may want to reference more than just images, should this be....

```suggestion
//= link_tree ../../../node_modules/identity-style-guide/dist/assets
```

Thinking specifically how we're referencing fonts in `identity-idp`.

If we update, we'll probably want to update the preceding text as well, since it mentions "images for production". "assets for production" might be more applicable in this case.

Reply:
Do you think we should reference the subdirectories explicitly? I'm thinking it might get confusing for Sprockets if it tries to precompile the SCSS and JS files separately from those parts of the asset pipeline.

Reply:
After looking at the `dist/assets` directory I think we should explicitly reference the fonts and images rather than the whole folder.

Reply:
Sounds good 👍  I agree with your point about JS and SCSS.